New version: RiskPerf v0.2.0

JuliaRegistrator opened the pull request:
- Registering package: RiskPerf
- Repository: https://github.com/rbeeli/RiskPerf.jl
- Created by: @rbeeli
- Version: v0.2.0
- Commit: 760e9d1b11ea02e56e23774d075557fec41847eb
- Reviewed by: @rbeeli
- Reference: https://github.com/rbeeli/RiskPerf.jl/commit/760e9d1b11ea02e56e23774d075557fec41847eb#commitcomment-162447337
- Description: Quantitative risk and performance analysis package for financial time series powered by the Julia language.
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## [0.2.0] – 2025‑07‑20

### Breaking changes

- N/A

### Changed

- Static exports list of package symbols instead of dynamic export for better static analysis and IDE integration
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6cf0275553ecee1f776112f758c71724d22405a181e602898f417237b38eb89976123a6e5a614227ade73fa3d8c2505390021b9f4100a3d574f062a75ce29319c9a5671c38dad35e4d42b9dfa8ddba926be5ff35c6d024ca293497c4338352c7ad38b852726ddc3c6f0c0d11ecb2fabf78b15805084700ae5326e8f36197b84387a8aebbe1cc8b7bb7e43a8d3248dc8201387bd48e5af8f13af7cadcf771d67dc990905cb8c44fa79ebd4e3316e2b7e3a -->

rbeeli commented:
Why does it think it's a breaking change release? Just went from 0.1.0 to 0.2.0?
[noblock]

JuliaRegistrator committed:
New version: RiskPerf v0.2.0

UUID: 9166ba07-fd5b-44b6-9936-7aa4238d27f8
Repo: https://github.com/rbeeli/RiskPerf.jl.git
Tree: 2a60b70493810745968f2eb075f22ac58bd488a2

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
